docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -342,14 +295,13 @@ you can only test via HTTP calls. Your test code does not actually run natively,
 that does not call your HTTP endpoints, it's probably not a good idea to run them as part of native tests.
 
 If you share your test class between JVM and native executions like we advise above, you can mark certain tests
-with the `@DisabledOnNativeImage` annotation in order to only run them on the JVM.
+with the `@DisabledOnIntegrationTest` annotation in order to only run them on the JVM.
 
 
 === Testing an existing native executable
 
 It is also possible to re-run the tests against a native executable that has already been built. To do this run
-`./mvnw test-compile failsafe:integration-test`. This will discover the existing native image and run the tests against it using
-failsafe.
+`./mvnw test-compile failsafe:integration-test`. This will discover the existing native image and run the tests against it using failsafe.
```

Comment:
I don't think you need `-Pnative` here. Maybe it should have been there before, since the failsafe config was only in the native profile?

Reply:
Ahh yes you are right. I see what you are getting at now. This is the building native image guide, so yes we will need to add the `-Pnative` flag.

But then that begs the question, shouldn't you always have had to include it, since the failsafe plugin itself was only in the `native` profile?

Reply:
I added `-Pnative` to the command in my latest commit.

Reply:
(I get confused if we should use the property or the profile.. I was pretty sure the property triggered the profile AND was used for something else.. )

Reply:
I always use the profile (not sure if that's what's preferred or not!)